blocks/cards/cards.css

```diff
@@ -22,109 +25,120 @@
   height: unset !important;      /* overwrite external CSS */
 }
 
-.franklin-card > a {
+.cards .card > a {
   display: block;
   width: 100%;
 }
 
-.franklin-card.with-border {
+.cards .card.with-border {
   box-shadow: 0 1px 6px 1px rgba(64 64 64 / 15%);
   border-radius: 8px;
   margin: 10px 0;
 }
 
-.franklin-card .card-image {
+.cards .card .card-image {
```

Comment:
The reason why I didn't namespace the `.card` with `.cards` is because we are reusing it in the carousel: https://github.com/hlxsites/walgreens/pull/40/files#diff-0ba135df53c420d893ac5474ae71abe07027087aba7d760a4534f1d585cefd4bR29 , to not duplicate the CSS.

(would need to be fixed in that PR from `.card` -> `franklin-card`)

Reply:
Let's not use the name "franklin".